index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
```

Comment:
Add responsive meta tags

Reply:
- [ ] Great job pointing out the changes required. However it's best to make it polite and friendly, instead of saying Add responsive meta tags, you can try Please add responsive meta tags... and give a reason for it. you can check out this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate).

Reply:
Please add responsive meta tag such as <meta name="viewport" content="width=device-width, initial-scale=1"> this means that the browser will (probably) render the width of the page at the width of its own screen.